projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.ts

```diff
@@ -31,8 +34,10 @@ export class RangeSliderComponent implements OnInit, AfterViewInit, OnDestroy, O
   @ViewChild('maxSlider')
   public maxSlider: ElementRef;
 
+  public _selectionChange: EventEmitter<SearchFacetSelection> = new EventEmitter<SearchFacetSelection>();
+
   @Output()
-  public selectionChange: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Drop a comment here on why this was needed!

Reply:
The emitter was emitting on every input change (slider movement).
I added a debounce as a temporary fix and have spoken with @teodossidossev about that.
A solution would be to emit on mouse up.

Reply:
I meant in the source code and not in the github :)

Reply:
I did add a comment in the code. Do you want it to be more descriptive?

Reply:
I saw it, it's OK.